sites/all/themes/auge-theme/sass/partials/components/panels/_frontpage.scss

```diff
@@ -64,6 +66,11 @@
       color: white;
     }
   }
+  &.panel-front-page-boxes-proximity {
+    a {
+      color: white;
```

Comment:
It's all the same so far as I know, the me from now would have prefered to use #fff but I'm just trying to make look a bit consistent with the old code.